### PR TITLE
[fix] fix err handler

### PR DESCRIPTION
**Pull request by donghui12**
**Please provide a description of this PR:**

[ ] Performance and Scalability
behavior changes
/kind bug

fix useless err

### CODE REVIEW

File: pilot/cmd/pilot-agent/app/cmd.go [hunk 1]
```diff
@@ -250,10 +250,7 @@ func getDNSDomain(podNamespace, domain string) string {
 }
 
 func configureLogging(_ *cobra.Command, _ []string) error {
-	if err := log.Configure(loggingOptions); err != nil {
-		return err
-	}
-	return nil
+	return log.Configure(loggingOptions)
```
Comment (hzxuzhonghu) [inline pilot/cmd/pilot-agent/app/cmd.go:253]: You can update all `configureLogging`

**Comment by donghui12** [comment]
Please help me add a label  release-notes-none ? \cc @ericvn

**Comment by donghui12** [comment]
> How does this fix anything?

**Comment by donghui12** [comment]
Remove useless if and err

**Pull request closed without merging**